truelearn/preprocessing/wikifier.py

```diff
@@ -0,0 +1,172 @@
+import orjson
+from urllib import request, parse, error
+
+from typing import Optional, Union, NoReturn
+Annotation = dict[str, Union[str, float, None]]
+WikifierResponse = dict[str, Union[list[str], list[dict]]]
+
+
+class Wikifier:
+    """

```

Comment:
We use a different style for docstring now. Please refer to [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html).

For most of your code, you need to use the `Args`, `Returns` and `Raises` specified in the guide.